### PR TITLE
Feature/hangout landing page

### LINKED ISSUE #46: View Hangout Information
When I view the list of hangouts, I want to be able to click on a hangout to see more information.

The additional information to be viewed for now, will be:
1. Hangout name
2. Date it's scheduled for

image

### LINKED ISSUE #47: Integrate hangout list with backend
Modify the hangout list to pull data from the backend service.

### PR DESCRIPTION
**Pull request by tiffanychenn**
resolved #46 and resolved #47

### CODE REVIEW

**Comment by tiffanychenn** [comment]
resolving #46 and resolving #47

**Comment by tiffanychenn** [comment]
Screen Shot 2019-06-24 at 1 23 09 PM
Screen Shot 2019-06-24 at 2 24 38 PM

**Comment by tiffanychenn** [comment]
currently the get request is in the FgHangouts page, should we take that out and put it in a service page instead?

**Comment by ArcticFaded** [comment]
Yes, basically in the hangouts service https://github.com/UltimateSoftware/fg-mobile/blob/master/src/domain/services/HangoutService.js, we want to fetch the data and commit it to the global state in services

when you write this function you would use something like hangoutActions.loadHangouts() in the componentDidMount, and for this issue since we want to use a service to perform the update we would require that the current component be a function and not a class (or else you wouldn't be able to take advantage of the React Hooks)

File: src/pages/FgHangouts.js [hunk 1]
```diff
@@ -1,37 +1,68 @@
...
+class FgHangouts extends React.Component{
+    constructor(props){
+        super(props);
+        this.state = {
+          hangouts: []
+        };
+    };
+
+    componentWillMount() {
+      fetch('http://ec2-34-201-168-0.compute-1.amazonaws.com:5000/hangouts', {
```
Comment (audgster) [inline src/pages/FgHangouts.js]: Convert to use a service

File: src/pages/HangoutDescription.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+/*
+ * FgHangouts might require a router of its own to create Icebreakers/games/etc.
+*/
+
+function HangoutDescription(nav){
+      const item = nav.navigation.getParam("item", "idk fam");
+      return(
+        <View style={{flex: 1}}>
+            <View style={{...styles.title, flex: 3}} >
+                <Text style={{...styles.title, color: 'white'}}>
```
Comment (audgster) [inline src/pages/HangoutDescription.js]: consider creating a component

**Pull request closed without merging**